batching/batching.go

```diff
@@ -275,6 +275,7 @@ func (b *Batcher) handleData(entry data.Entry) error {
 	batch, ok := b.current[entry.SourceType()]
 	if !ok {
 		batch = getBatch()
+		batch.Data = make(map[types.UID]data.Entry)
```

Comment:
Don't use make unless you are going to presize, instead use a composite literal aka: map[types.UID]data.Entry{}